aip/general/0193.md

```diff
@@ -86,13 +91,29 @@ if the resource or parent exists.
 If the user does have proper permission, but the requested resource or parent
 does not exist, the service **must** error with `NOT_FOUND` (HTTP 404).
 
+## Rationale
+
+`ErrorInfo` is required because it further identifies an error. The
+`Status` field, with under 20 [allowed values][Code], is rarley enough
+to diambiguate one error from another accross an entire API Service.
+Error messages often contain dynamic segments that express values, such
+as project numbers or locations, and these values are too often read
+from the string message using brittle extraction methods, like RegExp.
+There needs to be a machine readable component of *every* error response
+that enables clients to pull such information programatically--without
+parsing the string error message. For these reasons, `ErrorInfo` is
+required for *every* error response.
+
 ## Further reading
 
 - For which error codes to retry, see [AIP-194](https://aip.dev/194).
-- For how to retry errors in client libraries, see [AIP-4221](https://aip.dev/client-libraries/4221).
+- For how to retry errors in client libraries, see
+  [AIP-4221](https://aip.dev/client-libraries/4221).
 
 ## Changelog
 
+- **2023-05-04**: Require [`ErrorInfo`][ErrorInfo] for all error
```

Comment:
update to whenever you plan to merge this PR

Reply:
Done.